content/moredata/1.txt

```diff
@@ -18,7 +18,7 @@ type Genre {
 
 # Define Directives and index
 
-director.film: uid @reverse .
-genre: uid @reverse .
+director.film: [uid] @reverse .
+genre: [uid] @reverse .
```

Comment:
Some of this is covered here https://github.com/dgraph-io/tutorial/pull/53/files Not sure if it's okay to overwrite tho.

Reply:
Okay, I will be merging all the latest changes.

Reply:
so what have you guys decided? are we changing schema in this PR or in 53?